Classes/View/LegacyPreviewView.php

```diff
@@ -54,17 +54,17 @@ class LegacyPreviewView extends PreviewView {
     );
 
 
-    /**
+	/**
 	 * @codeCoverageIgnore
-     * @param array $row
-     * @param Column $column
-     * @param $colPosFluxContent
-     * @param $dblist
-     * @param $target
-     * @param $id
-     * @param $content
-     * @return string
-     */
+	 * @param array $row
+	 * @param Column $column
+	 * @param $colPosFluxContent
```

Comment:
What I mean is that the `@param $colPosFluxContent` should be `@param integer $colPosFluxContent`, `@param $dblist` should be `@param DatabaseRecordList $dblist` etc.